presto/eval.py

```diff
@@ -35,15 +35,14 @@
 class Hyperparams:
     lr: float = 2e-5
     max_epochs: int = 100
-    batch_size: int = 64
-    patience: int = 10
-    num_workers: int = 4
+    batch_size: int = 2048
```

Comment:
How were these chosen? This batch size is probably too large for finetuning

Reply:
indeed, reverted batch_size to 256 here https://github.com/WorldCereal/presto-worldcereal/commit/a350280be17c7715d4022ea78cd9f478e61bfa3c
I was playing with leaning rate, but also ended up using your value.